oaknational/Oak-Web-Application
# fix: downloads share url should point to lesson overview page

### Job Story

**When** using the share experiment from the download confirmation page
**I want the shared url to point to the lesson overview page**
**So that the other user arrives at an place that isn’t confusing**

### Implementation

Share sources schema (src=)

```tsx
  "lesson-browse": 0,
  "lesson-canonical": 1,
  "download-browse": 2,
  "download-canonical": 2,
  "lesson-listing": 3,
```

### Acceptance criteria

- [x]  the share url from download confirmation page points at the lesson overview page
- [x]  the browser url is unmodified (ie. no query params)
- [x]  share keys and ids are consistent between lesson overview and downloads pages
- [x]  share method  (sm=) is 1 for the button and 0 for url
- [x]  share source (src=) matches the above schema
## How to test

1. Go to https://deploy-preview-3034--oak-web-application.netlify.thenational.academy/teachers/lessons/transverse-waves/downloads
2. Click on the download resources button
3. You should see that the browser url remains unchanges
4. click on share lesson resources
5. you should see that the copied url has the correct parameters (cf. with the lesson overview page)
